### DISCUSSION TITLE
Chat Templates

### DISCUSSION THREAD

**Discussion by AWAS666** [PygmalionAI/aphrodite-engine#189] [General]
As far as I've checked, most hf models seems to be missing the chat template property in the tokenizer, even the pymaglion-2 one.
Is there an example of a working one?
No matter which one I tried, they all seemed to not stop at all and also output their formating tokens?

**Comment by AlpinDale** [comment]
There are a few templates in the `examples/` directory of this repo. You can also find more of them in [theroyallab/llm-prompt-templates](https://github.com/theroyallab/llm-prompt-templates).
